Modules/HR/Resources/views/application/follow_up.blade.php

```diff
@@ -0,0 +1,12 @@
+<div>
+    <p>Hello Pankaj Kandpal,</p>
```

Comment:
We need to pick the user name dynamic. Something like this.
```suggestion
    <p>Hello {{$user->name}},</p>
```